applications/GeoMechanicsApplication/custom_geometries/interface_geometry.h

```diff
@@ -280,7 +308,22 @@ class LineInterfaceGeometry : public Geometry<Node>
         return result;
     }
 
-    std::unique_ptr<BaseType> mMidLineGeometry;
+    [[nodiscard]] std::size_t GetNumberOfCornerPoints() const
+    {
+        const auto geometry_family = mMidGeometry->GetGeometryFamily();
+        switch (geometry_family) {
```

Comment:
Since the member function already clearly expresses what it returns, I feel that we don't really need to define the variable `geometry_family`:
```suggestion
        switch (mMidGeometry->GetGeometryFamily()) {
```

Reply:
Agreed, done